shell.nix

```diff
@@ -2,19 +2,12 @@
 # If you have nix installed, you may simply run `nix-shell`
 # in this repo, and have all dependencies ready in the new shell.
 
-{pkgs ? import (builtins.fetchTarball {
-  # Descriptive name to make the store path easier to identify
-  name = "nixos-2021-11";
-  # Commit hash
-  url = "https://github.com/NixOS/nixpkgs/archive/refs/tags/21.11.tar.gz";
-  # Hash obtained using `nix-prefetch-url --unpack <url>`
-  sha256 = "162dywda2dvfj1248afxc45kcrg83appjd0nmdb541hl7rnncf02";
-}) {}}:
-
-
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
This should be fine since I doubt any of these dependencies are changing in a meaningful way, as you see. Though it is not as robust to bitrot. Nix flakes should address the dependency pinning and composition problem a bit more robustly, and may be something to investigate in the near future.